doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -178,7 +178,7 @@ Cumulative, NonCumulative
 .. attr:: universes(cumulative{? = {| yes | no } })
    :name: universes(cumulative); Cumulative; NonCumulative
 
-   Polymorphic inductive types, coinductive types, variants and
+   Polymorphic inductive types, co-inductive types, variants and
```

Comment:
"coinduction" seems to be common in English without the dash: https://en.wikipedia.org/wiki/Coinduction

Reply:
I saw that, but I also saw `co-inductive` in http://adam.chlipala.net/cpdt/html/Coinductive.html.  Both forms are used.  Also, the existing text used `co-inductive` more often.

I'll change it if you prefer.

Reply:
> Also, the existing text used `co-inductive` more often.

That suffices to convince me.